README.md

```diff
@@ -1 +1,165 @@
-# rails-back-end-api
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] This repo is working well but we have requested changes because the frontend part is not approved. so when it will be approved your whole project will be approved 👍